specification/loadtestservice/LoadTestService/routes.tsp

```diff
@@ -280,11 +280,11 @@ numeric, underscore or hyphen characters.
 }
 
 interface LoadTestRun {
-  @summary("Get test run details by name.")
-  @doc("Get test run details by name.")
+  @summary("Delete a test run by its name.")
```

Comment:
Why did these operations get moved around? They look exactly the same as before except their positions are switched?

Reply:
Just revert this change and I don't think we need them now.